_pytest/pytester.py

```diff
@@ -908,8 +908,11 @@ def popen(self, cmdargs, stdout, stderr, **kw):
         env['PYTHONPATH'] = os.pathsep.join(filter(None, [
             str(os.getcwd()), env.get('PYTHONPATH', '')]))
         kw['env'] = env
-        return subprocess.Popen(cmdargs,
-                                stdout=stdout, stderr=stderr, **kw)
+
+        popen = subprocess.Popen(cmdargs, stdin=subprocess.PIPE, stdout=stdout, stderr=stderr, **kw)
```

Comment:
@RonnyPfannschmidt, I think, the final solution with the closed pipe is much better than the original, because this is tinier.

In other hands, the `DEVNULL` solution opened an unnecessary fd, but we wanted to make a not interactive `pdb` only.